README.md

```diff
@@ -94,10 +94,10 @@ To debug Unity games, you need this repo too: https://github.com/dnSpy/dnSpy-Uni
 
 # Wiki
 
-See the [Wiki](https://github.com/dnSpy/dnSpy/wiki) for build instructions and other documentation.
+See the [Wiki](https://github.com/dnSpyRevived/dnSpy/wiki) for build instructions and other documentation.
 
 # License
 
-dnSpy is licensed under [GPLv3](dnSpy/dnSpy/LicenseInfo/GPLv3.txt).
+dnSpy is licensed under [GPLv3](dnSpyRevived/dnSpy/LicenseInfo/GPLv3.txt).
```

Comment:
@Therzok FYI this link is now broken.

Reply:
Fixed in 30c758da09f3ff481034071f713c26bf27668b7e